obspy/io/sac/sactrace.py

```diff
@@ -459,10 +459,10 @@ def _strgetter(hdr):
     def get_str(self):
         try:
             # value is a bytes
-            value = self._hs[HD.STRHDRS.index(hdr)].decode()
+            value = unicode(self._hs[HD.STRHDRS.index(hdr)].decode())
         except AttributeError:
             # value is a str
-            value = self._hs[HD.STRHDRS.index(hdr)]
+            value = unicode(self._hs[HD.STRHDRS.index(hdr)])
```

Comment:
I'm in desperate need of a unicode/str/bytes/future education :smile: I was thinking of a separate PR to simplify encoding/decoding of characters in this subpackage: use only unicode internally, encode/decode at I/O time.  Good to know that `future` will interpret `str` for Python 2 vs 3.  I'll make that change.  By "eventually", you mean you intend to drop support for Python 2.7?